examples/Button.json

```diff
@@ -127,7 +127,39 @@
         }
     ],
     "computed": [],
-    "methods": [],
+    "methods": [
+        {
+            "keywords": [
+                {
+                    "name": "param",
+                    "description": "{string} question a question about life, the universe, everything"
+                },
+                {
+                    "name": "returns",
+                    "description": "{number} the answer to all your questions"
+                }
+            ],
+            "visibility": "public",
+            "description": "Computes the answer to your question.",
+            "name": "computeAnswer",
+            "args": [
+                {
+                    "name": "question"
+                }
+            ],
+            "return": {
+                "doctype": {
+                    "kind": "type",
+                    "text": "number",
+                    "type": "number"
+                },
+                "type": "number",
+                "desc": "the answer to all your questions",
```

Comment:
Not sure that we need to duplicate desciption for return. So, please keep only second property `description`